src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -2175,4 +2176,44 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
             return null;
         }
     }
+
+    /// <summary>
+    /// Converts element type of node search element in short string.
+    /// E.g. ElementTypes.Packaged => PKG.
+    /// </summary>
+    public class ElementTypeToShortConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var stringValue = value.ToString();
+            if (String.IsNullOrEmpty(stringValue))
+                throw new ArgumentException("Unknown element type.");
```

Comment:
The `value` is already of type `ElementTypes`, there's no need to turn it into `string` and then back to `ElementTypes` again :)